rest/model/host.go

```diff
@@ -62,18 +63,21 @@ func (apiOpts *APIProvisionOptions) BuildFromService(opts host.ProvisionOptions)
 // /hosts the yaml tags are used by hostCreate() when parsing the params from a
 // file.
 type HostRequestOptions struct {
-	DistroID              string     `json:"distro" yaml:"distro"`
-	TaskID                string     `json:"task" yaml:"task"`
-	TaskSync              bool       `json:"task_sync" yaml:"task_sync"`
-	Region                string     `json:"region" yaml:"region"`
-	KeyName               string     `json:"keyname" yaml:"key"`
-	UserData              string     `json:"userdata" yaml:"userdata_file"`
-	SetupScript           string     `json:"setup_script" yaml:"setup_file"`
-	UseProjectSetupScript bool       `json:"use_setup_script_path" yaml:"use_setup_script_path"`
-	Tag                   string     `yaml:"tag"`
-	InstanceTags          []host.Tag `json:"instance_tags" yaml:"instance_tags"`
-	InstanceType          string     `json:"instance_type" yaml:"type"`
-	NoExpiration          bool       `json:"no_expiration" yaml:"no-expire"`
+	DistroID              string            `json:"distro" yaml:"distro"`
+	TaskID                string            `json:"task" yaml:"task"`
+	ProjectRepo           string            `bson:"project_repo" json:"project_repo"`
```

Comment:
Do we need these if we have the task ID? (If we do, can we move them to the bottom with a comment about why we're adding them?)